requests/test/src/requests/RequestTests.scala

```diff
@@ -1,10 +1,21 @@
 package requests
 
+import java.net.InetSocketAddress
+
+import com.sun.net.httpserver.HttpExchange
+import com.sun.net.httpserver.HttpHandler
+import com.sun.net.httpserver.HttpServer
```

Comment:
Does this work on Java 11+ ?

Reply:
I thought I tested for all versions. I can try again ...